modules/bootkube/resources/bootkube.sh

```diff
@@ -25,15 +25,15 @@ echo "Rendering MCO manifests..."
 		--dest-dir=/assets/mco-bootstrap \
 		--images-json-configmap=/assets/manifests/machine-config-operator-01-images-configmap.yaml
 
-mkdir -p /etc/kubernetes/manifests/
-mkdir -p /etc/mcc/bootstrap/
-mkdir -p /etc/ssl/mcs/
+mkdir --parents /etc/kubernetes/manifests/
+mkdir --parents /etc/mcc/bootstrap/
+mkdir --parents /etc/ssl/mcs/
```

Comment:
I'm generally in favor of long optipns and spelling things out to make source more readable.  But [only `-p` is in POSIX][1], so I think we want to roll this particular change back for portability.

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/mkdir.html